src/agent/src/netlink.rs

```diff
@@ -0,0 +1,1020 @@
+use anyhow::Context;
```

Comment:
Copyright missing.

Reply:
Yep, I've updated the file with generic header (with no company name), just to pass checks and advance to unit tests. I'm still waiting for a legal approval.